Skeleton/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Having another package.json within the project won't break anything?

Reply:
it's ok to have a package.json on each folder you want to have special dependencies.
the resolver only search for node_modules by going up in the file tree